Research/ResearchWarcrimes.md

```diff
@@ -0,0 +1,41 @@
+<!-- It's not necessary to follow this format, as long as you provide a coherent and structured document -->
+
+## Abstract
+
+Research is currently suffering due to lack of care about it ever since the introduction of Techwebs/Removal of DEFCON. Currently it is functional at a minimum and with the lack of features, it rarely ever changes the round flow. First act of rejuvenation is including new features that researchers can do and provide more co-operation between OT and the marines.
```

Comment:
the abstract should be describing what research should be, not the current problems it faces